### PR TITLE
so share

### PR DESCRIPTION
**Pull request by defunctl**
Adds `so share` command to temporarily share your local project on the internet with Ngrok.

### CODE REVIEW

File: app/Commands/LocalDocker/Share.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+            $settings->ngrok_token = $authToken;
+            $settings->save();
+        }
+
+        $this->checkGitIgnore( $file, $config->getProjectRoot() );
+
+        $source  = storage_path( sprintf( 'wordpress/mu-plugins/%s', self::MU_PLUGIN ) );
+        $content = $filesystem->get( $source );
+        $target  = sprintf( '%s/%s', $config->getProjectRoot(), sprintf( 'wp-content/mu-plugins/%s', self::MU_PLUGIN ) );
```
Comment (jbrinley) [inline app/Commands/LocalDocker/Share.php:83]: Shouldn't affect many projects, but any with a different WP_CONTENT_DIR will have issues with this. Maybe doesn't matter. 🤷

**Review by jbrinley — APPROVED** [review]
This is great, @defunctl. Thanks!

There's some unrelated code in here related to autocompletion. Not sure if that was intentional or if it was meant to be in a separate PR.

**Comment by defunctl** [comment]
> This is great, @defunctl. Thanks!
> 
> There's some unrelated code in here related to autocompletion. Not sure if that was intentional or if it was meant to be in a separate PR.

Yeah I just threw it in to go out in the next release.

File: app/Commands/LocalDocker/Share.php (hunk 1 above)
Comment (defunctl) [reply app/Commands/LocalDocker/Share.php:83]: Good eye! I'll note for the future to add a configuration item to squareone.yml to configure this on a per project basis.

**Pull request merged**